course-definition.yml

```
@@ -4,8 +4,7 @@ short_name: "grep"
 release_status: "live"
 
 description_md: |-
-  Regular expressions (Regexes, for short) are patterns used to match character combinations in strings. In this
-  challenge you'll build your own implementation of grep, a CLI tool for searching using Regexes.
+  Regular expressions (or regex) are patterns used to match character combinations in strings. In this challenge, you'll build your own implementation of grep, a CLI tool for regex-based searching, without using any built-in regex features.
```

```suggestion
  Regular expressions (or regex) are patterns used to match character combinations in strings. In this challenge, you'll build a Regex engine from scratch by recreating grep, a CLI tool for regex-based searching.
```

(We don't have to say "without built-in regex features" in the description, only in stages where it matters)